scripts/core/services/pageTitle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PageTitle service
+ *
+ * PageTitle service is used to save and update the title of the page.
+ * The title will be 'Superdesk {- Url} {-Desk/Workspace}
+ * Url can be: Dashboard, Monitoring, Spike, Highlights, Search, Settings
+ * If Url is search then there won't be desk
+ *
+ */
+export default angular.module('superdesk.services.pageTitle', [])
+    .service('pageTitle', function() {
+        this.title = 'Superdesk';
+        this.url = '';
+        this.workspace = '';
+
+        this.setPageUrl = function(url) {
```

Comment:
I think `pageTitle.setPage*` is a bit stuttering. In my opinion it would be better to just call these methods `pageTitle.set*`.

Also, this is not really a URL, but more like a section. So would it not be better called simply `set`? Since this is a service, let's make it versatile and more global, so we can set any title we like.